lib/report.js

```diff
@@ -127,6 +129,29 @@ class Report {
     return sources
   }
 
+  /**
+   * //TODO: use https://www.npmjs.com/package/convert-source-map
```

Comment:
I'm perfectly comfortable with us floating our own version of this method, since we're currently only using one bit of the library (I'd rather not add an additional library).

Did we get this code directly from `convert-source-map`, because I think we should pull this into a `lib/` folder, carrying over the license header from that project.

Reply:
I think this is okay. The code isn't from `covert-source-map`, but after I wrote it I noticed we had `convert-source-map` in the dependency tree of `v8-to-istanbul` and were using it. So I thought we would be better served by using a function from there, but I don't feel strongly about it. I'll remove the TODO.